lib/logger/utils.ts

```diff
@@ -132,6 +133,12 @@ export default function prepareError(err: Error): Record<string, unknown> {
     response.errors = err.errors.map((error) => prepareError(error));
   }
 
+  // handle rawExec error
+  if (err instanceof ExecError && is.nonEmptyObject(err.options?.env)) {
+    const env = Object.keys(err.options.env);
+    response.options = { ...err.options, env };
```

Comment:
why not calling sanitizer on values only 🤔

Reply:
in the long living containers solutions we inherit the entire env from the parent, hence renovate is not able to santize renovate unrelated secrets.